armi/bookkeeping/db/tests/test_comparedb3.py

```diff
@@ -181,7 +181,7 @@ def test_compareDatabaseSim(self):
             dbs[1]._fullPath,
             timestepCompare=[(0, 0), (0, 1)],
         )
-        self.assertEqual(len(diffs.diffs), 504)
+        self.assertEqual(len(diffs.diffs), 576)
```

Comment:
This is a very big change. I am surprised.

Reply:
We chatted about this, most of these diffs are `None` diffs. I'm not sure I understand the point of that line in particular, it seems to be checking whether the PR is updating the number in accordance with the (new number of parameters on the block) * 3 since the diffs report 3 separate values for each diff (mean, median, max or something like that).

Reply:
I have rarely seen anyone's PR change this number by more than 1 or 2. 

So, I am suspicious that there is a problem here somewhere. My guess is this number will drop down a lot when the CI starts passing.